rest_framework/utils/model_meta.py

```diff
@@ -146,7 +152,11 @@ def _get_reverse_relationships(opts):
         )
 
     # Deal with reverse many-to-many relationships.
-    for relation in opts.get_all_related_many_to_many_objects():
+    if django.VERSION < (1, 9):
```

Comment:
It's probably better to use django version check instead of `getattr`. What do you think, should I change it?